tests/rust_reflection_test/src/main.rs

```diff
@@ -0,0 +1,286 @@
+use flatbuffers_reflection::reflection::*;
+use flatbuffers_reflection::*;
+
+use flatbuffers::*;
+
+use std::fs::File;
+use std::io::Read;
+
+#[allow(dead_code, unused_imports)]
+#[path = "../../monster_test/mod.rs"]
+mod monster_test_generated;
+pub use monster_test_generated::my_game;
+
+#[test]
+fn test_schema() {
```

Comment:
Would it make sense to split this test into smaller ones that check specific functionality?

Reply:
Done. Thanks!